### PR TITLE
add jfsutils

### PR DESCRIPTION
**Pull request by manveru**

### CODE REVIEW

File: srcpkgs/jfsutils/template [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+version=1.1.15
+revision=1
+build_pie=yes
+build_style=gnu-configure
+makedepends="util-linux e2fsprogs-devel"
+short_desc="JFS filesystem utilities"
+maintainer="Michael Fellinger <m.fellinger@gmail.com>"
+license="GPL"
+homepage="http://jfs.sourceforge.net"
+distfiles="http://jfs.sourceforge.net/project/pub/${pkgname}-${version}.tar.gz"
```
Comment (beefcurtains) [inline srcpkgs/jfsutils/template]: Use $SOURCEFORGE_SITE

File: srcpkgs/jfsutils/template [hunk 2]
```diff
@@ -0,0 +1,16 @@
+pkgname=jfsutils
+version=1.1.15
+revision=1
+build_pie=yes
+build_style=gnu-configure
+makedepends="util-linux e2fsprogs-devel"
```
Comment (xtraeme) [inline srcpkgs/jfsutils/template]: `util-linux` is wrong... it does not provide any shared library, I guess you want `libblkid-devel` or `libuuid-devel`.

File: srcpkgs/jfsutils/template [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+short_desc="JFS filesystem utilities"
+maintainer="Michael Fellinger <m.fellinger@gmail.com>"
+license="GPL"
```
Comment (xtraeme) [inline srcpkgs/jfsutils/template]: GPL is too vague... please specify the exact GPL version.

File: srcpkgs/jfsutils/template [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+homepage="http://jfs.sourceforge.net"
+distfiles="http://jfs.sourceforge.net/project/pub/${pkgname}-${version}.tar.gz"
+checksum=244a15f64015ce3ea17e49bdf6e1a0fb4f9af92b82fa9e05aa64cb30b5f07a4d
+
+pre_build() {
+  patch -p1 -i ${FILESDIR}/inttypes.patch
```
Comment (xtraeme) [inline srcpkgs/jfsutils/template]: This is really wrong, patches should be in `/patches` and is expected to be in `-p0` format.

**Comment by manveru** [comment]
Not going to force you to merge it, I just thought it might be useful, but don't have time to fix everything for a package i only use once a year.

**Comment by xtraeme** [comment]
Those tweaks can take you 5min of your life... really.

**Pull request closed without merging**